personal/privacyidea/class_mfaAccount.inc

```diff
@@ -974,6 +974,16 @@ class mfaAccount extends plugin
                 "mandatory"   => true,
                 "description" => _("privacyIDEA server URL.")
             ),
+            array(
+                "name"        => "piTokenOrigin",
+                "type"        => "string",
+                "check"       => "gosaProperty::isString",
+                "group"       => "core",
+                "default"     => "http://tjener.intern/",
```

Comment:
GOsa² != Debian Edu. Rather some generic thing: ```gosa.mysite.tld```

Reply:
This always requires adjustment, I guess.

Reply:
Please do not use `mysite.tld` but one of the reserved names `example.(com|net|org)` which exists for this purpose.